src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const lines = sourceString.split(';');
+  const result = {};
+
+  lines.forEach((line) => {
+    const trimmedLine = line.trim();
+
+    if (trimmedLine === '' || !trimmedLine.includes(':')) {
+      return;
+    }
 
+    const [property, ...valueParts] = trimmedLine.split(':');
+    const value = valueParts.join(':').trim();
+
+    if (property && value) {
+      result[property.trim()] = value.replace(/;$/, '').trim();
```

Comment:
The replace operation to remove the semicolon at the end of the value is redundant since the semicolon has already been used as the delimiter in the split operation on line 9.